docs/setup/installing/opensuse.rst

```diff
@@ -0,0 +1,50 @@
+.. _opensuse:
+
+.. toctree::
+
+Blog post
```

Comment:
Suggest 'See Also'

Reply:
Moved to the bottom of the page and called it "For more Information"